src/grt/src/cugr/src/Design.cpp

```diff
@@ -289,6 +302,34 @@ void Design::setUnitCosts()
   }
 }
 
+bool Design::isClkTerm(odb::dbITerm* iterm, sta::dbNetwork* network)
```

Comment:
This and others look like they are copied from GlobalRouter.  Is it possible to avoid copying?